src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/ExerciseInformationDTO.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.artemis.web.rest.dto.metrics;
+
+import java.time.ZonedDateTime;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.enumeration.DifficultyLevel;
+import de.tum.in.www1.artemis.domain.enumeration.ExerciseMode;
+import de.tum.in.www1.artemis.domain.enumeration.IncludedInOverallScore;
+
+/**
+ * DTO for exercise information.
+ *
+ * @param id                     the id of the exercise
+ * @param shortName              shortTitle the short title of the exercise
+ * @param title                  title the title of the exercise
+ * @param start                  the start date of the exercise
+ * @param due                    the due date of the exercise
+ * @param maxPoints              the maximum achievable points of the exercise
+ * @param includedInOverallScore whether the exercise is included in the overall score
+ * @param difficulty             the difficulty level of the exercise
+ * @param exerciseMode           the mode of the exercise
+ * @param type                   the type of the exercise
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record ExerciseInformationDTO(long id, String shortName, String title, ZonedDateTime start, ZonedDateTime due, Double maxPoints,
+        IncludedInOverallScore includedInOverallScore, DifficultyLevel difficulty, ExerciseMode exerciseMode, Class<? extends Exercise> type) {
```

Comment:
```suggestion
        IncludedInOverallScore includedInOverallScore, DifficultyLevel difficulty, ExerciseMode exerciseMode, ExerciseType type) {
```
Would the enum be better here?

Reply:
This is a complicated issue since fetching DTOs directly from the DB with such information is complex. This should be resolved Artemis wide. We created a issue to track it.